test/Microsoft.AspNetCore.SignalR.Tests/HubConnectionHandlerTests.cs

```diff
@@ -2181,6 +2181,15 @@ public void Release(THub hub)
             }
         }
 
+        public class AnotherTypedHub : Hub<IAnotherTypedHubClient>
+        {
+        }
+
+        public interface IAnotherTypedHubClient
+        {
+            void Foo();
```

Comment:
This will blow up after the other change 😄

Reply:
Yep, hoping to get that one in and then rebase this on top.